packages/knowledge/src/base.ts

```diff
@@ -1251,33 +1254,21 @@ export const base: ChartKnowledgeBase = {
     recRate: 'Recommended',
   },
 
-  polar_treemap: {
```

Comment:
what was this.....polar treemap....

Reply:
got it.....

https://g2.antv.vision/en/examples/relation/relation#polar-treemap

Reply:
I found the definition of a polar treemap on this site https://resources.wolframcloud.com/FunctionRepository/resources/PolarTreemapPlot/.

It looks the same as sunbrust, so I deleted the polar treemap.